pkg/pillar/cmd/zedrouter/pubsubhandlers.go

```diff
@@ -634,3 +634,27 @@ func (z *zedrouter) handleAppInstDelete(ctxArg interface{}, key string,
 	z.unpublishAppInstMetadata(appInstMetadata)
 	z.log.Functionf("handleAppInstDelete(%s) done", key)
 }
+
+func (z *zedrouter) handlePatchEnvelopeCreate(ctxArg interface{}, key string,
+	configArg interface{}) {
+}
+
+func (z *zedrouter) handlePatchEnvelopeModify(ctxArg interface{}, key string,
+	statusArg interface{}, oldStatusArg interface{}) {
+}
+
+func (z *zedrouter) handlePatchEnvelopeDelete(ctxArg interface{}, key string,
+	statusArg interface{}) {
+}
+
+func (z *zedrouter) handleVolumeStatusCreate(ctxArg interface{}, key string,
+	statusArg interface{}) {
+}
+
+func (z *zedrouter) handleVolumeStatusModify(ctxArg interface{}, key string,
+	statusArg interface{}, oldStatusArg interface{}) {
+}
+
+func (z *zedrouter) handleVolumeStatusDelete(ctxArg interface{}, key string,
+	statusArg interface{}) {
+}
```

Comment:
Are there functions supposed to be empty? If so you don't need to register functions in NewSubscription().

Reply:
Those functions are needed for external patch envelopes, removed declarations from this PR, will introduce them in separate PR

Reply:
ok